plenum/test/client/test_send_from_wrong_socket.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import zmq
+
+from plenum.test.client.helper import create_zmq_connection
+from plenum.test.logging.conftest import logsearch
+from stp_core.loop.eventually import eventually
+
+
+@pytest.fixture(params=[zmq.REQ])
```

Comment:
Can we check other socket types? SUB, PUB, ROUTER, etc.

Reply:
For other socket types we will have another behaviour. All of this cases will cause to ZMQErrors.